python/runtime/model/metadata_test.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from runtime.feature.column import NumericColumn
+from runtime.feature.field_desc import FieldDesc
+from runtime.model.metadata import (collect_metadata, load_metadata,
+                                    save_metadata)
+
+FILENAME = 'meta.json'
```

Comment:
Shall we use FILE_NAME instead of FILENAME?

Reply:
Done. Move to `self.file_name`.